config/schedule.rb

```diff
@@ -0,0 +1,24 @@
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# Example:
+#
+# set :output, "/path/to/my/cron_log.log"
+#
+# every 2.hours do
+#   command "/usr/bin/some_great_command"
+#   runner "MyModel.some_method"
+#   rake "some:great:rake:task"
+# end
+#
+# every 4.days do
+#   runner "AnotherModel.prune_old_records"
+# end
+
+every 1.day, at: "8:30 am" do
```

Comment:
Is there some way we can get this to retry if the service fails for whatever reason? Or perhaps that doesn't matter because we can run the service again manually if we see an entry in Sentry?

Reply:
Aye... I originally planned to run it in a sidekiq worker so as we would get the retries. I decided not to as:

* it will get called 3 times anyway from each backend box
* I thought we'd probably want to know and intervene for now rather than have retries queueing up in sidekiq
* it takes a bit of load of the queues (and inversely doesn't have to wait if the queue is busy)

All that said, I'm happy to run it in sidekiq if we decide that would be better